src/Yoti/Util/Profile/AnchorListConverter.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace Yoti\Util\Profile;
+
+use Traversable;
+
+class AnchorListConverter
+{
+    public static function convert(Traversable $anchorList)
+    {
+        $yotiAnchorsMap = [];
+
+        foreach ($anchorList as $protobufAnchor) {
+            if ($parsedAnchor = AnchorConverter::convert($protobufAnchor)) {
+                $yotiAnchorsMap[$parsedAnchor['oid']][] = $parsedAnchor['yoti_anchor'];
```

Comment:
Will this allow us to have multiple sources and multiple verifiers with the same oid (key) for an attribute?

Reply:
Yes and there is a test case for that [here](https://github.com/getyoti/yoti-php-sdk/blob/SDK-659/tests/Util/Profile/AnchorListConverterTest.php#L25)

Reply:
👍